src/containers/Login/LoginForm.tsx

```diff
@@ -24,19 +25,30 @@ export const FormStyled = styled(Form)`
 export const LoginForm = () => {
   return (
     <Formik
-      initialValues={ {
+      initialValues = {{
         email: '',
         password: '',
-      } }
-      onSubmit={ () => {
-      } }
+      }}
+      onSubmit={ (values) => {} }
```

Comment:
values prop is redundant here for now
let us add it during the bl implementation